python/ray/rllib/dqn/base_evaluator.py

```diff
@@ -55,20 +55,23 @@ def update_target(self):
         self.dqn_graph.update_target(self.sess)
 
     def sample(self):
-        output = []
+        obs, actions, rewards, new_obs, dones = [], [], [], [], []
```

Comment:
Does it make sense to replace this with a `SyncSampler`? A little modification would need to be made to support exploration, but this class will end up cleaner. Can be done in another PR I guess

Reply:
Sure, I think it should work.